Samples/Samples/ViewModel/CalendarViewModel.cs

```diff
@@ -11,28 +13,77 @@ public class CalendarViewModel : BaseViewModel
         public CalendarViewModel()
         {
             GetCalendars = new Command(OnClickGetCalendars);
+            GetAllEvents = new Command(OnClickGetEvents);
             RequestCalendarReadAccess = new Command(OnRequestCalendarReadAccess);
             RequestCalendarWriteAccess = new Command(OnRequestCalendarWriteAccess);
         }
 
         public ICommand GetCalendars { get; }
 
+        public ICommand GetAllEvents { get; }
+
         public ICommand RequestCalendarReadAccess { get; }
 
         public ICommand RequestCalendarWriteAccess { get; }
 
         public ObservableCollection<ICalendar> Calendars { get; } = new ObservableCollection<ICalendar>();
 
+        public ObservableCollection<IEvent> Events { get; } = new ObservableCollection<IEvent>();
+
+        ICalendar selectedCalendar;
+
+        public ICalendar SelectedCalendar
+        {
+            get
+            {
+                return selectedCalendar;
+            }
+
+            set
+            {
+                if (selectedCalendar != value)
+                {
+                    selectedCalendar = value;
+                    OnPropertyChanged("SelectedCalendar");
```

Comment:
use nameof(SelectedCalendar)
better still - if the base viewmodel has it, use SetProperty

Reply:
Yep, fixed